stackit/internal/services/loadbalancer/loadbalancer/resource.go

```diff
@@ -642,31 +671,42 @@ func (r *loadBalancerResource) Update(ctx context.Context, req resource.UpdateRe
 	ctx = tflog.SetField(ctx, "project_id", projectId)
 	ctx = tflog.SetField(ctx, "name", name)
 
-	for _, targetPool := range model.TargetPools {
+	targetPools := []targetPool{}
```

Comment:
In some of the other fields, you call this variable  <field>Model but here it doesn't have the "Model" suffix

I would actually remove the "Model" suffix in the other variable names, seems redundant

Reply:
I've renamed this variable to `targetPoolModel`, let me know if you want me to revert this and rename the other `...Model` variables

Reply:
fine for me